src/i18n/it-IT.json

```
@@ -59,7 +59,8 @@
         "group_created_description": "Gruppo creato con successo",
         "email": "Indirizzo mail",
         "user_mail": "Indirizzo mail",
-        "password_expired_tooltip": "Password scaduta"
+        "password_expired_tooltip": "Password scaduta",
+        "invalid_username": "Sono consentiti solo caratteri alfanumerici minuscoli e trattini bassi"
```

Maybe "underscore" is more technically correct

```suggestion
        "invalid_username": "Sono consentiti solo caratteri alfanumerici minuscoli e underscore"
```